CHANGELOG.md

```diff
@@ -17,6 +17,10 @@
 * [CHANGE] Blocks uploaded by ingester no longer contain `__org_id__` label. Compactor now ignores this label and will compact blocks with and without this label together. `mimirconvert` tool will remove the label from blocks as "unknown" label. #1972
 * [CHANGE] Querier: deprecated `-querier.shuffle-sharding-ingesters-lookback-period`, instead adding `-querier.shuffle-sharding-ingesters-enabled` to enable or disable shuffle sharding on the read path. The value of `-querier.query-ingesters-within` is now used internally for shuffle sharding lookback. #2110
 * [CHANGE] Memberlist: `-memberlist.abort-if-join-fails` now defaults to false. Previously it defaulted to true. #2168
+* [FEATURE] Ingester: Add experimental ability to ingest out of order samples up to an allowed limit. Enabling this takes additional memory and disk space. It also enables a write behind log that could lead to longer ingester start replays. There is no overhead on memory, disk space, startup times, with it being disabled. #2187
+  * `-ingester.out-of-order-allowance` allows setting how back in time a sample can be. Defaults to `0`.
+  * `-blocks-storage.tsdb.out-of-order-cap-min` sets the minimum capacity of the out-of-order chunk allocated in the memory. Defaults to `4`
```

Comment:
`-blocks-storage.tsdb.out-of-order-cap-min` and `-blocks-storage.tsdb.out-of-order-cap-max` are very low level settings, not sure it needs a call out in changelog. On the other hand, I don't see new metrics mentioned.

Reply:
If we do not require mentioning all flags in the changelog, then yes, let's remove it from here because a user does not require to know about this. I can remove it. Will mention metrics.